Userland/Libraries/LibVideo/VP9/Decoder.h

```diff
@@ -138,9 +153,16 @@ class Decoder {
         Vector<Intermediate> transform_temp;
         Vector<i64> adst_temp;
 
+        // predict_inter
+        Vector<u16> inter_horizontal;
+        Vector<u16> inter_predicted;
+        Vector<u16> inter_predicted_compound;
+
         Vector<Intermediate> intermediate[3];
         Vector<u16> output[3];
     } m_buffers;
+
+    bool m_enable_debug { false };
```

Comment:
Is this (still) used?

Reply:
Ah, I guess I didn't manage to avoid staging this field one of the many times I had to add debug printing. Good catch, I've removed that.